src/features/left-sidebar/LeftSidebar.js

```diff
@@ -18,6 +18,7 @@ import NewItemsIndicator from './NewItemsIndicator';
 import type { Callout } from './Callout';
 
 import './styles/LeftSidebar.scss';
+import FooterIndicator from '../../components/footer-indicator/FooterIndicator';
```

Comment:
Can you move this up to its friends so the SCSS import can be last?

Reply:
@jstoffan Done!